snmp/datadog_checks/snmp/data/default_profiles/dell.yaml

```diff
@@ -1,15 +0,0 @@
-# Profile for Dell  devices
-
-extends:
-  - _base.yaml
-  - _generic-ip.yaml
-  - _generic-tcp.yaml
-  - _generic-udp.yaml
-
-metadata:
-  device:
-    fields:
-      vendor:
-        value: "dell"
-
-sysobjectid: 1.3.6.1.4.1.674.*
```

Comment:
I think we should probably keep the dell.yaml for now with initial content:

```
# Profile for Dell  devices

extends:
  - _dell.yaml
  - _generic-ip.yaml
  - _generic-tcp.yaml
  - _generic-udp.yaml

sysobjectid: 1.3.6.1.4.1.674.*
```

I think it can be a breaking change to remove them.

Reply:
Just saw https://github.com/DataDog/integrations-core/pull/15446/commits/904459875eeff24437ca9b404a952083c3f24ac6

Can we make `dell.yaml` import `_dell.yaml` ? :)